package.json

```diff
@@ -24,14 +36,26 @@
   },
   "homepage": "https://github.com/ethereumjs/ethereumjs-account#readme",
   "dependencies": {
-    "ethereumjs-util": "^5.0.0",
-    "rlp": "^2.0.0",
+    "bn.js": "^4.11.8",
```

Comment:
Where is this new `bn.js` dependency coming from respectively used?

Reply:
I had mistakenly thought this is also necessary for the error (see below). Removed it now.